app/helpers/administrate/application_helper.rb

```diff
@@ -4,5 +4,17 @@ def render_field(field, locals = {})
       locals.merge!(field: field)
       render locals: locals, partial: field.to_partial_path
     end
+
+    def display_resource_name(resource_name)
+      resource_name.
```

Comment:
Yeah, I'm siding with @mcmire on this. I've been using `resource` to refer to instances of the ActiveRecord models. Here, we're referring to the class names of the ActiveRecord models.

I think if we end up going with @mcmire's suggestion in https://github.com/thoughtbot/administrate/pull/298#discussion_r46846324, then we should rename this to `resource_class` or something.